Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -87,29 +88,56 @@ class TabManagerImplementation: LegacyTabManager {
     // MARK: - Save tabs
 
     override func preserveTabs() {
-        guard AppConstants.useNewTabDataStore else {
-            super.preserveTabs()
+        /// For now we want to continue writing to both data stores so that we can revert to the old system if needed
```

Comment:
Otherwise triggers the orphaned doc comment rule
```suggestion
        // For now we want to continue writing to both data stores so that we can revert to the old system if needed
```